tests/FSharp.Compiler.Service.Tests/FSharp.CompilerService.SurfaceArea.netstandard.expected

```diff
@@ -3654,8 +3654,8 @@ FSharp.Compiler.EditorServices.Structure+Scope: Boolean Equals(System.Object, Sy
 FSharp.Compiler.EditorServices.Structure+Scope: Boolean IsArrayOrList
 FSharp.Compiler.EditorServices.Structure+Scope: Boolean IsAttribute
 FSharp.Compiler.EditorServices.Structure+Scope: Boolean IsComment
-FSharp.Compiler.EditorServices.Structure+Scope: Boolean IsCompExpr
 FSharp.Compiler.EditorServices.Structure+Scope: Boolean IsCompExprInternal
```

Comment:
As before should we also rename IsCompExprInternal --> IsComputationExprInternal